### PR TITLE
Add support for SPAN

### PR DESCRIPTION
**Pull request by RunDevelopment**
This adds support for [SPAN](https://github.com/hongyuanyu/SPAN) as requested by @kim2091. No model tests this time, because the official models are all in one 1.2GB .zip file, and I don't want our poor CI to download that. I tested the 4 official models and they all work fine.

Parameter detection is fairly trivial this time, because there are only 4 params that matter and can be detected. As always, I cannot test FP16.

### CODE REVIEW

File: src/spandrel/architectures/SPAN/__init__.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        rgb_mean=rgb_mean,
+    )
+
+    return ImageModelDescriptor(
+        model,
+        state_dict,
+        architecture="SPSR",
+        purpose="Restoration" if upscale == 1 else "SR",
+        tags=[f"{feature_channels}nf"],
+        supports_half=True,
```
Comment (joeyballentine) [inline src/spandrel/architectures/SPAN/__init__.py:40]: Always assume this is false until it can be tested pls

Comment (Kim2091) [reply src/spandrel/architectures/SPAN/__init__.py:40]: I'll test this when it's implemented in chaiNNer-nightly

File: src/spandrel/architectures/SPAN/arch/LICESNE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (joeyballentine) [inline src/spandrel/architectures/SPAN/arch/LICESNE:1]: This file is spelled wrong

Comment (RunDevelopment) [reply src/spandrel/architectures/SPAN/arch/LICESNE:1]: Indeed it is.

**Review by joeyballentine — APPROVED** [review]

**Pull request merged**